node/src/accountant/scanners/mod.rs

```diff
@@ -905,39 +924,62 @@ impl ReceivableScanner {
         }
     }
 
-    fn handle_new_received_payments(&mut self, msg: &ReceivedPayments, logger: &Logger) {
-        let mut txn = self
-            .receivable_dao
-            .as_mut()
-            .more_money_received(msg.timestamp, &msg.payments);
-
-        let new_start_block = msg.new_start_block;
-        match self
-            .persistent_configuration
-            .set_start_block_from_txn(new_start_block, &mut txn)
-        {
-            Ok(()) => (),
-            Err(e) => panic!(
-                "Attempt to set new start block to {} failed due to: {:?}",
-                new_start_block, e
-            ),
-        }
+    fn handle_new_received_payments(
+        &mut self,
+        payments_and_start_block: &PaymentsAndStartBlock,
+        timestamp: SystemTime,
+        logger: &Logger,
+    ) {
+        if payments_and_start_block.payments.is_empty() {
+            info!(
+                logger,
+                "No newly received payments were detected during the scanning process."
+            );
+            let new_start_block = payments_and_start_block.new_start_block;
+            match self
+                .persistent_configuration
+                .set_start_block(new_start_block)
+            {
+                Ok(()) => debug!(logger, "Start block updated to {}", new_start_block),
+                Err(e) => panic!(
+                    "Attempt to set new start block to {} failed due to: {:?}",
+                    new_start_block, e
+                ),
+            }
+            debug!(logger, "Updated start block to: {}", new_start_block)
```

Comment:
This debug repeats the one being in the OK branch of the match statement above it.